doc/source/examples/multiphysics/water-injection-in-a-closed-cell/water-injection-in-a-closed-cell.rst

```diff
@@ -61,7 +61,7 @@ The initial time step is set to :math:`0.005 \, \text{s}` and the simulation end
       set time end         = 0.49
```

Comment:
line 25 :  replace  "Python script for postprocessing:" by "Postprocessing python script:" for uniformizing the notations with the other examples